app/modules/auth/models.py

```diff
@@ -311,6 +294,16 @@ def generate(cls, length=8):
         new_code = ''.join(new_code)
         return new_code
 
+    @classmethod
+    def valid_codes(cls, user, code_type):
+        now = datetime.datetime.now(tz=pytz.utc)
```

Comment:
We have a os.environ.get('TZ') but I don't know if that should be used here

Reply:
I've seen code using `current_app.config.get('TIMEZONE')` but I'm mostly following the original `is_expired` code:

https://github.com/WildMeOrg/houston/blob/1e604118a753c21772fab719d6c71f05f031956a/app/modules/auth/models.py#L489-L493

The "expires" column in the database is datetime without timezone...  and since we're comparing dates maybe it's better to just always use utc?  The `TZ` in the environment can change and so can the app config one... :thinking: